adapter/redisquota/redisquota_test.go

```diff
@@ -421,3 +475,97 @@ func TestBadConfig(t *testing.T) {
 	}
 
 }
+
+func TestErrorResponse(t *testing.T) {
+	mockredis := testServer{errOnStart: false, errOnIntRet: true}
+	_, f, _ := mockredis.Start()
```

Comment:
this should probably be guarded. race condition testing seems to cause issues with multiple attempts to bind to the same port.